data/eurovis-2019/script/.gitignore

```diff
@@ -0,0 +1,3 @@
+connections.csv
+Eurovis2019Network.json
+people.csv
```

Comment:
Unless these files are very big (which in this case, I can't imagine they are), my practice has been to simply run the processing script and check these result files into GitHub alongside the raw data. The processing script would be there largely as an indicator of provenance for the resulting data files.

So to be explicit: I would remove this .gitignore file, and add the three files it references.

Reply:
These files are checked in at `data/eurovis-2019/connections.csv` and `data/eurovis-2019/people.csv`. I copied your format of moving them up a level in the directory structure and out of the scripts folder.

Reply:
Oh gotcha. That's so that if you run the scripts here, you don't see untracked files showing up in your git repo. I'm not sure my earlier practice was great, but since you've remained consistent with it, I'm fine with this approach.